src/test/java/com/firebolt/jdbc/util/StringUtilTest.java

```diff
@@ -0,0 +1,62 @@
+package com.firebolt.jdbc.util;
+
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class StringUtilTest {
+    @ParameterizedTest
+    @CsvSource(value = {
+            ",",
+            "'',''",
+            "1,1",
+            "true,true",
+            "'hello',hello"
+    }, delimiter = ',')
+    void stripSingleQuotes(String value, String expected) {
+        assertEquals(expected, StringUtil.strip(value, '\t'));
+    }
```

Comment:
How does this test work, we're stripping `\t`, a tab character, and not quotes and is still succeeds?

Reply:
You are absolutely right. It seems that I was drunk ;) while coding this test. Fixed.

Reply:
Lol. I'm just surprised this test run at all.